content/docs/reference/builder-config.md

```diff
@@ -28,6 +28,23 @@ A [builder][builder] configuration schema is as follows:
     <br>
     A URL or path to an [archive](#supported-archives), or a path to a directory. If path is relative, it must be relative to the `builder.toml`.
 
+- #### `packages` _(list, optional)_
+  <br>
+
+  A list of packaged buildpacks, each with the following fields:
+
+  - **`id`** _(string, optional)_
+    <br>
+    as defined above.
+
+  - **`version`** _(string, optional)_
+    <br>
+    as defined above.
+
+  - **`image`** _(string, required)_
+    <br>
+    A registry location (if no registry host is specified in the image name, DockerHub is assumed).
+
 - #### `order` _(list, required)_
   <br>
 
```

Comment:
We should add a reference doc for `package.toml` as part of this story similar to this page.

Reply:
Adding this.